### ISSUE TITLE
Quantify IR for single end bam files

### ISSUE THREAD

**Issue by me37uday** [williamritchie/IRFinder#144]
Hi @dg520 ,

Hope you are doing well :)

I was wondering if there is any pre-processing that needs to be done before quantifying introns in single end forward stranded bam files. Please do let me know about it.  

I had to ask because when I tried to quantify for unsorted single end bam file, I had Low Coverage for majority of the intronic regions in spite of sufficient sequencing depth (170 million reads). Please let me know your opinion on the same.

Cheers,
Uday

**Comment by dg520** [comment]
@me37uday No, there is no specific pre-processing needed for single-end data. Low Coverage for majority of intronic region is totally expected. Or are you talking about the low coverage warning in the last column? That is NOT for intronic region, but for the expected splicing flanking the intron instead. If that's the case, I recommend you to investigate your alignment first.
